src/store/actions.js

```diff
@@ -19,3 +19,8 @@ export const changeSystemColumns = (payload) =>({
     type: 'CHANGE_SYSTEM_COLUMNS',
     payload
 });
+
+export const loadSuggestedInstanceTypes = (params) =>({
+    type: 'LOAD_ROS_SYSTEMS',
```

Comment:
@r14chandra can we use some other name here please which will be relevant to suggested instances types?